brax/training/agents/ppo/train.py

```diff
@@ -218,14 +226,16 @@ def train(
           randomization_fn, rng=randomization_rng
       )
     if isinstance(environment, envs.Env):
-      wrap_for_training = envs.training.wrap
+      wrap_for_training = functools.partial(
```

Comment:
ah i see

did you try instead changing the order of the wrapping in wrap_for_training such that VmapWrapper is created first?  Then you might not need this scan param